src/translations/en.json

```diff
@@ -1355,8 +1355,8 @@
                 "metric_example": "Celsius, kilograms",
                 "find_currency_value": "Find your value",
                 "save_button": "Save",
-                "external_url": "Internet",
-                "internal_url": "Local network",
+                "external_url": "External URL",
+                "internal_url": "Internal URL",
```

Comment:
?

Reply:
They are not used in that form, or probably anywhere else.